libs/blocks/marquee/marquee.js

```diff
@@ -78,6 +78,7 @@ export async function loadMnemonicList(foreground) {
   }
 }
 
+
```

Comment:
@Axelcureno can you fix this eslint issue. Looks like this file shouldn't even be in the change list?

Reply:
Fixed